app/digests/api.py

```diff
@@ -46,6 +46,16 @@ def _create_response(self, data: Dict[str, Any]) -> Response:
     def _can_preview(self) -> bool:
         return self.request.META.get(settings.AUTHORIZATION_PREVIEW_HEADER, False)
 
+    @staticmethod
+    def _filter_by_related_content_id(qs: QuerySet, content_id: str) -> QuerySet:
+        """Will filter by the first `relatedContent` `id` value if present.
+
+        :param qs: class: `QuerySet`
+        :param content_id: str
+        :return: class: `QuerySet`
+        """
+        return qs.filter(relatedContent__0__id=content_id)
```

Comment:
given we are now assuming a single `relatedContent` item, should we add a validation for it in the write side that does not allow more than 1 item when creating/editing a digest?

Reply:
It uses the schema for validation, so if it's enforced in that, it will automatically be enforced here.

Though, from previous discussions, the `relatedContent` rarely does, but can, contain multiple items. Checking the first item was agreed as a sensible default to start with.

Reply:
Features team seems to have decided this will always be 1 item, but the schema is wider than that. Could restrict the schema for now?